std/process.d

```diff
@@ -1443,6 +1499,33 @@ int wait(Pid pid) @safe
     else version (Posix) assert(pid.osHandle < 0);
 }
 
+@system unittest // Pid.fromSystemPid()
+{
+    version (Windows)    TestScript prog = "do { timeout 1 } while (1)";
```

Comment:
What the heck is this syntax? It's definitely not `.cmd` syntax. Is this a broken test?